rs3-chunkpicker-chunkinfo-export.json

```diff
@@ -165855,7 +165914,7 @@
 				"1": "Always"
 			}
 		},
-		"Automaton (Heart of Gielinor)": {
+		"Automaton (HoG)": {
```

Comment:
My personal preference would be not to use an abbreviation here (abbreviations are confusing). [Wiki also refers to them in full.](https://runescape.wiki/w/Automaton_(Heart_of_Gielinor)) Similar discussion to be had about "(GWD)"; OSRS picker doesn't abbreviate it, so I'd go for the option that preserves parity as well.

This is a bigger thing to unravel because all of the Heart of Gielinor monster drops have (HoG) appended, except the monsters inside the Heart chunk *don't* actually refer to a "(HoG)" variant aside for Chaos dwarves.

Reply:
I changed from HoG to full name for Automaton (distinction matters as these don't count for slayer tasks)
Will discuss further for renaming the rest both for HoG and GWD